examples/deployment/docker/db_client/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.8
+
+RUN apt-get update && \
+    apt-get install -y mysql-client
+
+ADD . /go/src/github.com/google/trillian
+WORKDIR /go/src/github.com/google/trillian
+
+RUN go get -v ./server/trillian_log_server
```

Comment:
removed this line but going to keep the source code included so `/scripts/*.sh` can be run